doc/examples/visualization.ipynb

```diff
@@ -66,23 +67,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Custom labels\n",
-    "labels = {}\n",
-    "for model in models:\n",
-    "    labels[model.hash] = \"M_\" + \"\".join(\n",
-    "        \"1\" if value == petab_select.ESTIMATE else \"0\"\n",
-    "        for value in model.parameters.values()\n",
-    "    )\n",
-    "labels[VIRTUAL_INITIAL_MODEL.hash] = \"\\n\".join(\n",
-    "    petab_select.VIRTUAL_INITIAL_MODEL.model_subspace_id.split(\"_\")\n",
-    ").title()\n",
-    "\n",
     "# Custom colors for some models\n",
     "colors = {\n",
-    "    \"M_000\": \"lightgreen\",\n",
-    "    \"M_001\": \"lightgreen\",\n",
+    "    models[\"M-000\"].hash: \"lightgreen\",\n",
+    "    models[\"M-001\"].hash: \"lightgreen\",\n",
```

Comment:
Optional: It might be convenient if one could alternatively provide this info as `model_hash_string => color`. `PlotData` has all the information available to convert the input to `hash => color `

Reply:
Thanks, I forgot to change that, it also works with `model_hash_string`